docs/encyclopedia/error-handling.mdx

```diff
@@ -103,7 +103,7 @@ function submitTransaction(tx, timeout) {
 
 </CodeExample>
 
-We assume the existence of a sleep implementation similar to the one [here](https://stackoverflow.com/a/39914235) Be sure to integrate backoff into your retry mechanism. In our example error-handling code above, we implement a simple linear backoff, but there are [plenty of recommendations](https://backoff-utils.readthedocs.io/en/latest/strategies.html#why-are-backoff-strategies-useful) for various other strategies. Backoff is important both for maintaining performance and avoiding rate-limiting issues.
+We assume the existence of a sleep implementation similar to the one [here](https://stackoverflow.com/a/39914235) Be sure to integrate back-off into your retry mechanism. In our example error-handling code above, we implement a simple linear back-off, but there are [plenty of recommendations](https://backoff-utils.readthedocs.io/en/latest/strategies.html#why-are-backoff-strategies-useful) for various other strategies. Back-off is important both for maintaining performance and avoiding rate-limiting issues.
```

Comment:
@ElliotFriend what do you think of this one?

Reply:
Looks like `backoff` is the more common usage across various projects/products. I think we can remove this suggestion.

Reply:
This has now been reverted.  Thanks!